addon/components/model-form.hbs

```diff
@@ -3,9 +3,38 @@
   @on-submit={{perform this.submit}}
   @validateBeforeSubmit={{true}}
   class="validated-form"
-  ...attributes
-  as |f|
+  ...attributes as |f|
 >
+
+  <ModelForm::Header 
+    @model={{@headerModel}}
+    @modelId={{@headerModelId}}
+    @projectId={{@projectId}}
+    @modelName={{@headerModelName}}
+    @modelDescription={{@headerModelDescription}}
+    @hasNoStatus={{@hasNoStatus}}
+    @hasImport={{@hasImport}}
```

Comment:
Maybe unify the names here a bit. Either go with `hasNoStatus` and `hasNoImport` or `hasStatus` and `hasImport`